backend/lib/services/terminals/index.js

```diff
@@ -56,16 +56,47 @@ const TargetEnum = {
   SHOOT: 'shoot'
 }
 
-exports.create = function ({ user, namespace, name, target, body = {} }) {
+exports.create = function ({ user, coordinate: { namespace, name, target }, body = {} }) {
```

Comment:
I find the term `coordinate` not intuitive. It's part of the body. On the next level it's gone. Why do we introduce it only here?

Reply:
we agreed to leave coordinate for now as I did not found any better term.